pages/process/[id]/hierarchy/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { ButtonNavigateToProcess } from "@/components/ButtonNavigateToProcess";
+import { ProcessHierarchyTabs } from "@/components/canvas/ProcessHierarchy/ProcessHierarchyTabs";
+import { useProjectId } from "@/hooks/useProjectId";
+import { Layouts } from "@/layouts/LayoutWrapper";
+import { getGraph } from "@/services/graphApi";
+import { getProject } from "@/services/projectApi";
+import { CircularProgress } from "@equinor/eds-core-react";
+import Head from "next/head";
+import { useQueries } from "react-query";
+import styles from "./hierarchy.module.scss";
+
+export default function HierarchyPage() {
+  const { projectId } = useProjectId();
+
+  const [projectQuery, graphQuery] = useQueries([
+    {
+      queryKey: ["project", projectId],
+      queryFn: () => getProject(projectId),
+      enabled: !!projectId,
+    },
+    {
+      queryKey: ["graph", projectId],
+      queryFn: () => getGraph(projectId),
+      enabled: !!projectId,
+    },
+  ]);
```

Comment:
I see both `getGraph` and `getProject` being used multiple places across the application. One problem with this approach is that you have to have steady hands when entering queryKeys wherever these functions are used.

To make it simpler to maintain, I would suggest creating two different hooks `useProjectQuery` and `useGraphQuery`, and use those across the application instead:
```ts
export const useProjectQuery = (projectId?: string) => useQuery({
    queryKey: ["project", projectId],
    queryFn: () => getProject(projectId),
    enabled: !!projectId,
})
```

```ts
export const useGraphQuery = (projectId?: string) => useQuery({
    queryKey: ["graph", projectId],
    queryFn: () => getGraph(projectId),
    enabled: !!projectId,
})
```